repository/t.json

```diff
@@ -3246,6 +3246,17 @@
 				}
 			]
 		},
+		{
+			"name": "Trusted Coding",
+			"details": "https://bitbucket.org/madnext/trustedcodingsublime",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": true,
+					"platforms": "*"
```

Comment:
If platforms is "*" please leave it out.

Reply:
Ok! Removed in the new commit